### PR TITLE
Delete VPC peering connections

### PR DESCRIPTION
**Pull request by aledbf**
## Description

Delete existing peering connections instead of failing VPC deletion

## TODOs

Read the [Gruntwork contribution guidelines](https://gruntwork.notion.site/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e).

- [ ] Update the docs.
- [ ] Run the relevant tests successfully, including pre-commit checks.
- [X] Include release notes. If this PR is backward incompatible, include a migration guide.
- [ ] Attention Grunts - if this PR adds support for a new resource, ensure the `nuke_sandbox` and `nuke_phxdevops` jobs in `.circleci/config.yml` have been updated with appropriate exclusions (either directly in the job or via the `.circleci/nuke_config.yml` file) to prevent nuking IAM roles, groups, resources, etc that are important for the test accounts.

## Release Notes (draft)

Delete VPC peering connections

### CODE REVIEW

File: aws/ec2.go [hunk 1]
```diff
@@ -768,3 +775,33 @@ func GetEC2ResourceNameTagValue(tags []*ec2.Tag) (string, error) {
 	}
 	return "", fmt.Errorf("Resource does not have Name tag")
 }
+
+func (v Vpc) nukePeeringConnections() error {
+	peeringConnections, err := v.svc.DescribeVpcPeeringConnections(
```
Comment (ellisonc) [inline aws/ec2.go:780]: This function will only return the first page of peering connections. In the (admittedly unlikely) event that there is more than a single page, errors will occur. Can you swap this out with the paginated version of this function? [aws docs](https://docs.aws.amazon.com/sdk-for-go/api/service/ec2/#EC2.DescribeVpcPeeringConnectionsPages)

**Review by ellisonc — CHANGES_REQUESTED** [review]
Could you please add a test case to `ec2_vpc_test.go` that exercises your new functionality?

File: aws/ec2.go (hunk 1 above)
Comment (aledbf) [reply aws/ec2.go]: done

**Comment by hongil0316** [comment]
Hi @aledbf, do you plan to continue making changes to this PR? I see that the branch has conflicts to resolve. Can you make necessary changes and let us know when it's ready? If I do not hear back for a while, I'll close this PR and we can re-open it later time when you have free time to continue the contribution.

**Comment by hongil0316** [comment]
Closing this PR since we haven't heard back for quite some time. Feel free to re-open the PR when necessary.

**Pull request closed without merging**